src/main/java/net/mcreator/element/ModElementType.java

```diff
@@ -164,4 +165,7 @@ public boolean hasProcedureTriggers() {
 	public static ModElementType<?> TAG;
 	public static ModElementType<?> TOOL;
 
+	// legacy
+	public static final ModElementType<?> FOOD = new ModElementType<>("food", null, BaseType.ITEM, RecipeType.ITEM, null, Food.class); // we keep it with its values, so we can make a basic converter.

```

Comment:
You should still init it as other elements, but mark it deprecated and annotated needed references with suppressed warnings